hips/draw/simple.py

```diff
@@ -83,108 +100,53 @@ def dst(self) -> np.ndarray:
              [0, 0]],
         )
 
-    @property
-    def projection(self) -> ProjectiveTransform:
+    def projection(self, tile: HipsTile) -> ProjectiveTransform:
         """Estimate projective transformation on a HiPS tile"""
-        corners = self.tile.meta.skycoord_corners.to_pixel(self.geometry.wcs)
+        corners = tile.meta.skycoord_corners.to_pixel(self.geometry.wcs)
         src = np.array(corners).T.reshape((4, 2))
         dst = self.dst
         pt = ProjectiveTransform()
         pt.estimate(src, dst)
         return pt
 
-    def warp_image(self) -> np.ndarray:
+    def fetch_tiles(self) -> 'HipsTile':
+        """Fetch HiPS tiles from a remote URL"""
+        for healpix_pixel_index in self.tile_indices:
+            tile_meta = HipsTileMeta(
+                order=self.hips_order,
+                ipix=healpix_pixel_index,
+                frame=self.hips_survey.astropy_frame,
+                file_format=self.tile_format,
+            )
+            tile = HipsTile.fetch(tile_meta, self.hips_survey.tile_access_url(order=self.hips_order, ipix=healpix_pixel_index) + tile_meta.filename)
+            yield tile
+
+    @property
+    def shape(self) -> tuple:
+        """Shape of the output image"""
```

Comment:
Would this be a good place to document what the shape will be for the three format cases?
Or can you add this info what shape computed images have somewhere else in the docs?